dist/images/Dockerfile.fedora

```diff
@@ -9,25 +9,94 @@
 # are built locally and included in the image (instead of the rpm)
 #
 
-FROM fedora:41
+ARG OVN_FROM=koji
+
+FROM fedora:41 AS ovnbuilder
```

Comment:
This will be run all the time, regardless of `OVN_FROM` value, increasing build time.

Reply:
OK.  Cool!